srcpkgs/antimicrox/template

```diff
@@ -12,23 +12,11 @@ maintainer="fosslinux <fosslinux@aussies.space>"
 license="GPL-3.0-or-later"
 homepage="https://github.com/AntiMicroX/antimicrox"
 distfiles="${homepage}/archive/${version}.tar.gz"
-checksum=53c6b0e6c8ea8c3a5527b56e1ad052eae983ba36abfc090769c6164bc5f342dd
+checksum=65fac9a57bc2c070241d0dc3e777f44cdecce96a667ac51bf89f04ceeb302c55
 
-libantimicrox_package() {
-	short_desc+=" - libraries"
-	pkg_install() {
-		vmove usr/lib/*.so.*
-	}
-}
-
-libantimicrox-devel_package() {
-	depends="libantimicrox>=${version}_${revision}"
-	short_desc+=" - development files"
-	pkg_install() {
-		vmove usr/lib/*.so
-		vmove usr/include
-	}
-}
+case "$XBPS_TARGET_MACHINE" in
```

Comment:
You could probably just use `XBPS_TARGET_LIBC`.

Reply:
yes, true:
```
if [ "$XBPS_TARGET_LIBC" = "musl" ]; then
inner stuff
fi
```
is better